### PR TITLE
ui admin

### PR DESCRIPTION
**Pull request by anhcnn-0378**
## Related Tickets:
- [#68187](https://edu-redmine.sun-asterisk.vn/issues/68187)

## WHAT this PR does?
- Tạo trang hiển thị list user cho admin (route chưa có middleware check role admin)

## Notes Impacted Areas:
- N/A.

## Attachments:
image

### CODE REVIEW

**Comment by anhcnn-0378** [comment]
ready

File: resources/views/layouts/admin.blade.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+                        <!-- Add more sidebar links as needed -->
+                    </ul>
+                </div>
+            </aside>
+
+            <div style="display:inline-block;padding-left:220px">
+                <!-- Page Heading -->
+                <header class="bg-white shadow max-w-7xl">
+                    <div class="max-w-7xl mx-auto py-6 px-4 sm:px-6 lg:px-8">
+                        {{ $header ?? ''}}
```
Comment (ghost) [inline resources/views/layouts/admin.blade.php]:
```suggestion
                        {{ $header ?? '' }}
```

File: resources/views/layouts/admin.blade.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="csrf-token" content="{{ csrf_token() }}">
+
+        <title>{{ config('app.name', 'Laravel') }}</title>
+
+        <!-- Fonts -->
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Nunito:wght@400;600;700&display=swap">
```
Comment (ghost) [inline resources/views/layouts/admin.blade.php]: download font, tránh link online

File: resources/views/layouts/admin.blade.php (hunk 1 above)
Comment (anhcnn-0378) [reply resources/views/layouts/admin.blade.php]: Em đã sửa ạ

File: resources/views/layouts/admin.blade.php (hunk 2 above)
Comment (anhcnn-0378) [reply resources/views/layouts/admin.blade.php]: Em đã sửa ạ

**Comment by anhcnn-0378** [comment]
ready

**Pull request merged**